README.md

```diff
@@ -1 +1,83 @@
-# Event-Finder
+# Event Finder
+
+> Tge Event Finder application is an appliation where you can find A list of events with details about each event. 
+- Display a list of Event.
+- Filter Events By Category (Music, Sports).
+- See Details about a specific Event.
+
+## Built With
+
+- React
+- Html
+- Css
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+Copy the SSH code to download all the files.
+
+### Prerequisites
+
+- Github.
+- Code editor.
+- Git Bash
+- Node Js
+
+#### Setup
+
+Open Git Bash and run
+`git clone https://github.com/desorgui/event-finder.git`
+
+#### Install
+
+Run `npm install` in the folder **_event-finder_**
+
+#### Test
+
+Run 
+`npx stylelint "**/*.{css,scss}"` to test css Linter
+`npx eslint .` to test Javascript Linter
+`npx hint .` to test html Linter
+
+#### Usage
+
+To run the server use, `npm start`
+
+Runs the app in the development mode.\
+Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+
+The page will reload when you make changes.\
+You may also see any lint errors in the console.
+
+`npm test`
+
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+
+`npm run build`
+
+Builds the app for production to the `build` folder.\
+It correctly bundles React in production mode and optimizes the build for the best performance.
+
+The build is minified and the filenames include the hashes.\
+Your app is ready to be deployed!
+
+## Authors
+
+👤 **Guishny Desor**
+
+- GitHub: https://github.com/desorgui
+- Twitter: https://twitter.com/DGuishny
+- LinkedIn:https://www.linkedin.com/in/guishny-desor-5421a01a9/
+
+Contributions, issues, and feature requests are welcome!
+
```

Comment:
- [ ] please kindly add an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).